whipper/common/mbngs.py

```diff
@@ -254,7 +295,13 @@ def _getMetadata(release, discid=None, country=None):
                     track.title = t.get('title', t['recording']['title'])
                     track.mbid = t['id']
                     track.mbidRecording = t['recording']['id']
-                    track.mbidWorks = _getWorks(t['recording'])
+                    works = _getWorks(t['recording'])
+                    track.mbidWorks = _getWorksId(works)
+                    if classical:
+                        track.composers = _getComposers(works)
+                        track.performers = _getPerformers(
+                            t['recording']['artist-relation-list']
+                             )
```

Comment:
If we’re getting the data for these two either, we may as well populate them regardless of classical mode. Classical mode might be useful for falling back on populating the values with Track and Recording Artist information though when the relationships are not available.

Maybe something like `if classical and len(track.composers) == 0: track.composers = [track.artist]` (and same for performers)?

Reply:
Yeah i just thought it would bloat metadata up. Making it mandatory should make the arguments not get thrown around code base. Thanks :D

Reply:
I think this should be good now. Thanks !